hazelcast/src/main/java/com/hazelcast/map/impl/querycache/QueryCacheEventService.java

```diff
@@ -49,16 +48,17 @@
      * @param listenerAdapter listener adapter for the query-cache
      * @return ID of registered event listener
      */
-    String listenPublisher(String mapName, String cacheId, ListenerAdapter listenerAdapter);
+    String addPublisherListener(String mapName, String cacheId, ListenerAdapter listenerAdapter);
 
     /**
      * Removes listener from underlying IMap
      *
      * @param mapName    underlying map name which query cache listens
+     * @param cacheId    id of the query cache
```

Comment:
Please use `ID` instead of `id` (see documentation style guide)

Reply:
fixed